k8s/src/main/scala/io/buoyant/k8s/v1.scala

```diff
@@ -96,7 +96,7 @@ package object v1 {
       HighResTimer.Default,
       stats,
       RetryBudget.Infinite
-    )
+    ) andThen new LogFailureFilter(log, stats)
```

Comment:
Actually I just noticed that there's already some logging in the `infiniteRetryFilter`'s predicate.  I removed `LogFailureFilter` entirely.